src/main/resources/default/taglib/k/base.html.pasta

```diff
@@ -108,44 +108,53 @@
             </i:if>
         </i:block>
 
-        <div class="row">
-            <div class="col-xl-9 col-lg-12">
-                <i:render name="body"/>
+        <t:sidebar renderSidebarIfEmpty="false" class="sticky-sidebar sticky-top mb-4">
+            <i:local name="sidebarContents" value="@renderToString('sidebar')"/>
+
+            <i:block name="sidebar">
+                <i:render name="sidebar"/>
+            </i:block>
+
+
+            <div class="row">
+                <div class="@if (isFilled(sidebarContents)) { col } else { col-xl-9 col-lg-12 }">
+                    <i:render name="body"/>
+                </div>
             </div>
-        </div>
 
-        <div class="row">
-            <div class="col-xl-9 col-lg-12">
-                <i:local name="references" value="article.queryCrossReferences()"/>
-                <i:if test="!references.isEmpty()">
-                    <k:section heading="@i18n('KnowledgeBase.crossReferencesLabel')">
-                        <i:for type="sirius.biz.tycho.kb.KnowledgeBaseArticle" var="reference" items="references">
-                            <div class="pb-2">
-                                <div>
-                                    <a href="/kba/@reference.getLanguage()/@reference.getArticleId()">
-                                        <i:if test="reference.isChapter()">
-                                            <i class="fa fa-book"></i>
-                                            <i:else>
-                                                <i class="fa fa-file-alt"></i>
-                                            </i:else>
-                                        </i:if>
-                                        @reference.getTitle()
-                                    </a>
+            <div class="row">
+                <div class="@if (isFilled(sidebarContents)) { col } else { col-xl-9 col-lg-12 }">
```

Comment:
**Question:** Do we want this? This would result in quite long text lines in the KB articles, which means the text is harder to read.

Reply:
Since we render a sidebar to the left if there are any `sidebarContents` the size of the KB articles remains roughly the same. Also we did the same thing in the `API`-Docs... just wanted to make it equal

Reply:
With sidebar:
![Bildschirmfoto 2023-11-10 um 11 40 59](https://github.com/scireum/sirius-biz/assets/106659278/af787032-bf32-486f-a06d-95f61a9769f7)

Without sidebar:
![Bildschirmfoto 2023-11-10 um 11 41 21](https://github.com/scireum/sirius-biz/assets/106659278/4e618427-65f8-4c66-b78c-eb768723cd7a)

Reply:
Thats why screenshots on PRs are very helpful 👍🏻